rdr_service/offline/export_va_workqueue.py

```diff
@@ -0,0 +1,369 @@
+import os
+import datetime
+
+from rdr_service import clock, config
+from rdr_service.api_util import list_blobs, delete_cloud_file
+from rdr_service.offline.sql_exporter import SqlExporter
+from rdr_service.dao.hpo_dao import HPODao
+from rdr_service.dao.participant_summary_dao import ParticipantSummaryDao
+
+CSV_HEADER = ['PMI ID',
+              'Biobank ID',
+              'Last Name',
+              'First Name',
+              'Middle Initial',
+              'Age Range',
```

Comment:
Thanks for getting that unit test set up. The only other thing I'm seeing that might need to be updated is that the date of birth should be included in the file output (both here in the headers, and down where the rows are written to the file).

Reply:
It should be. I was looking at to_client_json and thought dateOfBirth was being replaced with ageRange because of the way it handles all of the datetime fields. I've added it in now.